iOS_SDK/OneSignalSDK/OneSignalUser/Source/OSUserRequests.swift

```diff
@@ -269,6 +266,9 @@ class OSUserExecutor {
         }
 
         OneSignalClient.shared().execute(request) { response in
+            // Clear local data in preparation for hydration
+            OneSignalUserManagerImpl.sharedInstance.clearUserData()
```

Comment:
Should this be inside the`if let response = response` block? What would it mean to have a nil response?

Reply:
Yes good call! It makes more sense inside the block. I have no idea what it would mean to have a successful yet `nil` response. Better to not make any local changes in that case.